src/interfaces/IMorpho.sol

```diff
@@ -112,6 +112,8 @@ interface IMorphoBase {
     /// on `transfer` and `transferFrom`. In particular, tokens with fees on transfer are not supported.
     /// - The IRM should not re-enter Morpho.
     /// - The oracle should return a price with the correct scaling.
+    /// - The oracle should not be able to go up instantly by a multiplier of more than 1/(LLTV*LIF). In particular, if
+    /// the loan asset is a vault, the oracle should not price its shares using the AUM.
```

Comment:
```suggestion
    /// the loan asset is a MetaMorpho vault, the oracle should not price its shares using the AUM.
```

or

```suggestion
    /// the loan asset is a vault that can receive donations, the oracle should not price its shares using the AUM.
```

4626 vaults in general could be donation resistant and exclude donations from AUM in the sense of `totalAssets`.